TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -4,15 +4,15 @@
     private static var configured = false
     public static let launcher = TradeItLauncher()
 
-    private static var _linkedBrokerManager: TradeItLinkedBrokerManager?
+    internal static var _linkedBrokerManager: TradeItLinkedBrokerManager?
```

Comment:
Had to make these internal so that the `@testable` tag will allow access. I think this is bad too. We should discuss workaround.